### DISCUSSION TITLE
Imposto de PJ's no Simples Nacional 2018 - 6% ou 15,5% na NF?

### DISCUSSION THREAD

**Discussion by andrearanha** [frontendbr/forum#1313] [Discussão]
Olá pessoal,

Quem trabalha como PJ deve saber que no início de 2018 as regras do Simples Nacional mudaram.
Em suma, um imposto de 6% mudou para 15,5%!!! Apesar de ter algumas brechas para pagar os 6%.

Gostaria de saber se vocês todos conseguiram voltar a pagar 6%, ou estão pagando 15,5% ainda.
Eu e meu contador achamos duas formas de ficar nos 6, legalmente, mas vi muita gente xingando por aí.

**Comment by RicardoWEBSiTE** [comment]
Fala Andre pode explicar a forma que vcs encontraram para permanecer nos 6%
pois os 15% é uma interpletacao de uso intelectual, ai da esse pulo para
15% de imposto, meu contador me alertou e teria que assumir a culpa caso a
receita verificar o tipo de serviço prestado. Qualquer ajuda é bem vinda
valeu.

Em qui, 13 de dez de 2018 às 17:03, André Aranha <notifications@github.com>
escreveu:

**Comment by andrearanha** [comment]
> Fala Andre pode explicar a forma que vcs encontraram para permanecer nos 6% pois os 15% é uma interpletacao de uso intelectual, ai da esse pulo para 15% de imposto, meu contador me alertou e teria que assumir a culpa caso a receita verificar o tipo de serviço prestado. Qualquer ajuda é bem vinda valeu. Em qui, 13 de dez de 2018 às 17:03, André Aranha <notifications@github.com> escreveu:
> […](#)
> Olá pessoal, Quem trabalha como PJ deve saber que no início de 2018 as regras do Simples Nacional mudaram. Em suma, um imposto de 6% mudou para 15,5%!!! Apesar de ter algumas brechas para pagar os 6%. Gostaria de saber se vocês todos conseguiram voltar a pagar 6%, ou estão pagando 15,5% ainda. Eu e meu contador achamos duas formas de ficar nos 6, legalmente, mas vi muita gente xingando por aí. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#1313>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA41Uns7h4ypJqIvTVQN01gcKe8NyHKYks5u4qRugaJpZM4ZSO5Z> .

Olá @RicardoWEBSiTE ,
Realmente, faturar numa atividade diferente do que vc faz tem um risco.
Mas mesmo nas atividades intelectuais, é legalmente possível pagar 6%. Aqui explica como: https://www.contratopj.com.br/como-nao-pagar-15-5-no-fator-r-anexo-iii-simples-nacional-2018/

Mas enfim, você tá pagando 15,5% até hoje??? :open_mouth:

**Comment by RicardoWEBSiTE** [comment]
Valeu Andre pelo link ira ajudar muito, mas nao eu abri um novo CNPJ pois
era MEI e assumi o risco, mas vou usar essa estrategia para ficar esperto,
valeu pia por compartilhar.

Em sáb, 15 de dez de 2018 às 12:12, André Aranha <notifications@github.com>
escreveu:

**Comment by andrearanha** [comment]
Pessoal, boa noite,

Montei uma calculadora CLT x PJ bem diferente do que tem por aí na internet.
Antes de publicar, porém, eu gostaria da opinião de vocês dessa thread. Por favor, me digam o que acham, se está fácil de entender, etc.

Valeu gente.

[Calculadora CLT x PJ by ContratoPJ.xlsx](https://github.com/frontendbr/forum/files/3494757/Calculadora.CLT.x.PJ.by.ContratoPJ.xlsx)
